provider/aws/elbv2.go

```diff
@@ -0,0 +1,272 @@
+package aws
+
+import (
+	"encoding/json"
+	"errors"
+	"finala/config"
+	"finala/expression"
+	"finala/storage"
+	"finala/structs"
+	"regexp"
+	"strings"
+	"time"
+
+	awsClient "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/pricing"
+	log "github.com/sirupsen/logrus"
+)
+
+// ELBClientDescreptor is an interface defining the aws elbv2 client
```

Comment:
Please change the `ELBClientDescreptor` to the name of the function

Reply:
Updated